project/plugins.sbt

```diff
@@ -4,3 +4,4 @@ addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.2")
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.11")
 addSbtPlugin("com.47deg"  % "sbt-microsites" % "0.9.0")
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.3.0")
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.16")
```

Comment:
So sorry Chris, according to https://scalameta.org/scalafmt/docs/installation.html the standard is 
`addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.0")`
which I think explains why you got so many changes when you ran the formatter. Would you mind adding that plugin instead and reformatting?

So sorry again! I should have committed this way before.